showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/models/Item.java

```diff
@@ -9,13 +10,14 @@
 
 @Entity
 @Table(name="Items")
+@JsonIgnoreProperties({"hibernateLazyInitializer","handler"})
 public class Item {
     @Id
     @GeneratedValue(strategy= GenerationType.IDENTITY)
     private Integer id;
 
     @JsonIgnore
-    @ManyToOne(fetch=FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
nitpick (so feel free to ignore): the above argument for name inside the Tables annotation doesn't have spaces before and after the equal sign, so i would just make sure styling is consistent here